examples/connect_ecat_coe.py

```diff
@@ -0,0 +1,42 @@
+from ingeniamotion import MotionController
+
+
+def connect_ethercat_coe(interface_index: int, slave_id: int, dictionary_path: str) -> None:
+    mc = MotionController()
+
+    interface_list_human_format = mc.communication.get_interface_name_list()
+    print("List of interfaces - Human-readable format:")
```

Comment:
I'd remove the "Human-readable format"

Reply:
It is an example, not a script. Add prints or logs could be good, but sometimes adding comments in the code is better than adding prints. No changes are required, but I think it is good to remember for future examples.